src/components/build_meta.js

```diff
@@ -12,6 +13,7 @@ class BuildMeta extends React.Component {
         <em>{build.author}</em>
         <span>pushed to</span>
         <em>{build.branch}</em>
+        <Timestamp time={build.created_at || build.enqueued_at} format="full" className="time"/>
```

Comment:
we've been using `react-timeago` to display the build date in the user interface. Example at https://github.com/drone/drone-ui/blob/master/src/components/build_panel.js#L39:L51

We might want to do the same here for consistency